identifiers/country-kr.csv

```diff
@@ -18,10 +18,12 @@ ocd-division/country:kr/ed:bucheon_city_2,Bucheon City Second
 ocd-division/country:kr/ed:bucheon_city_3,Bucheon City Third
 ocd-division/country:kr/ed:bucheon_city_fourth,Bucheon City Fourth
 ocd-division/country:kr/ed:buk_district,Buk District
-ocd-division/country:kr/ed:buk_district_1,Buk District First
-ocd-division/country:kr/ed:buk_district_2,Buk District Second
 ocd-division/country:kr/ed:buk_district_and_gangseo_district_1,Buk District and Gangseo District First
 ocd-division/country:kr/ed:buk_district_and_gangseo_district_2,Buk District and Gangseo District Second
+ocd-division/country:kr/ed:bukd_district_1,Buk District First (Daegu)

```

Comment:
instead of attaching a letter to the end of the first word in the district name, what if we include the city/province name in the front of the district ID. So in this case it'd be: `ed:daegu_buk_district_1`

Reply:
Done